drf_recaptcha/fields.py

```diff
@@ -82,3 +82,14 @@ def __init__(self, action: str, required_score: float = None, **kwargs):
             secret_key=settings.DRF_RECAPTCHA_SECRET_KEY,
         )
         self.validators.append(validator)
+
+    @property
+    def score(self):
+        score = self.validators[-1].score
```

Comment:
may be will better like that?
self.__validator = ReCaptchaV3Validator(
self.validators.append(self.__validator)
```suggestion
        score = self.__validator.score
```

Reply:
I wasn't super please using the positional index for the validator. This is much nicer solution.